source/jsonnet/england-wales/lib/placeholders.libsonnet

```diff
@@ -95,53 +97,33 @@ local firstPersonNamePossessiveForList(listName) = {
       },
       transform: 'concatenate_list',
     },
-    {
-      transform: 'format_possessive',
-      arguments: {
-        string_to_format: {
-          source: 'previous_transform',
-        },
-      },
-    },
+    transforms.formatPossessive,
   ],
 };
 
+local personName(sameNameTransform='') = (
+  if sameNameTransform == 'contains_same_name_items' then
+    {
+      placeholder: 'person_name',
+      transforms: [transforms.containsSameNameItems, transforms.formatFirstPersonName],
+    }
+  else if sameNameTransform == 'list_has_same_name_items' then
+    {
+      placeholder: 'person_name',
+      transforms: [transforms.listHasSameNameItems, transforms.formatFirstPersonName],
+    }
+  else
+    {
+      placeholder: 'person_name',
+      transforms: [transforms.concatenateList],
```

Comment:
Is there a reason not to use the `format_name` transform with `include_middle_names` set to `false`?

Reply:
I was thinking the same, it was how we wanted to deliver removing concatenate_list. Usually we would do the feature, then the schema, or both at the same time. This effectively would be a schema change then, raising a card for runner

Reply:
Why does it need a runner change? `concatenate_list` is still a valid transform, it just wouldn't be used for this.